web/app/js/components/EdgesTable.jsx

```diff
@@ -82,21 +83,21 @@ const edgesColumnDefinitions = (PrefixedLink, namespace, type, classes) => {
   ];
 };
 
-const generateEdgesTableTitle = edges => {
+const generateEdgesTableTitle = (edges, t) => {
   let title = 'Edges';
   if (edges.length > 0) {
     let identity = edges[0].direction === 'INBOUND' ? edges[0].serverId : edges[0].clientId;
     if (identity) {
       identity = `${identity.split('.')[0]}.${identity.split('.')[1]}`;
-      title = `${title} (Identity: ${identity})`;
+      title = `${title} (${t('Identity')}: ${identity})`;
```

Comment:
As this can change depending on the language, we'll want to use [interpolation](https://www.i18next.com/translation-function/interpolation) here. That'll allow folks to l10n with the order of their own language!